ui/client/components/graph/node-modal/editors/expression/ExpressionSuggester.js

```diff
@@ -207,12 +207,34 @@ export default class ExpressionSuggester {
       const properties = this._alreadyTypedProperties(currentProjectionOrSelection)
       //TODO: currently we don't assume nested selections/projections
       const focusedClazz = this._findRootClazz(properties, this._variables)
-      return (focusedClazz || {}).params ? focusedClazz.params[0] : null
+      return this._getFirstParameterType(focusedClazz)
     } else {
       return null
     }
   }
 
+  _getFirstParameterType = (typ) => {
```

Comment:
We have v. nice unit tests for this class, can you add one more? :)

Reply:
Test added